sim/paladin/avengers_shield.go

```diff
@@ -31,14 +31,18 @@ func (paladin *Paladin) registerAvengersShieldSpell() {
 				damage := 1100.0 + deltaDamage*sim.RandomFloat("Damage Roll")
 				damage += hitEffect.SpellPower(spell.Unit, spell) * scaling.SP
 				damage += hitEffect.MeleeAttackPower(spell.Unit) * scaling.AP
+				damage *= core.TernaryFloat64(paladin.HasMajorGlyph(41101), 2, 1)
```

Comment:
This doesnt have to be a dynamic multiplier as it never changes. Cache the value of HasMajorGlyph so you can reuse it in the next call and move this multiplir to the SpellEffect.DamageMultiplier